src/Microsoft.ML.LightGBM/LightGbmCatalog.cs

```diff
@@ -14,20 +14,20 @@ namespace Microsoft.ML
     public static class LightGbmExtensions
     {
         /// <summary>
-        /// Predict a target using a decision tree regression model trained with the <see cref="LightGbmRegressorTrainer"/>.
+        /// Predict a target using a boosted decision tree regression model trained with the <see cref="LightGbmRegressorTrainer"/>.
```

Comment:
> a boosted decision tree re [](start = 34, length = 27)

"gradient boosting decision tree" to be consistent with other docs.

Reply:
done

---
In reply to: [263946628](https://github.com/dotnet/machinelearning/pull/2886#discussion_r263946628) [](ancestors = 263946628)